frontend/src/features/mapSideBar/SidebarContents/subforms/LandSearchForm.tsx

```diff
@@ -0,0 +1,163 @@
+  );
+  return (
+    <Row noGutters className="section">
+      <Col md={12}>
+        <h5>Search for Parcel</h5>
+      </Col>
+      <Col md={6}>
+        <Form.Row>
+          <Label>PID</Label>
+          <Input
+            displayErrorTooltips
+            className="input-small"
+            disabled={false}
+            pattern={RegExp(/^[\d\- ]*$/)}
+            onBlurFormatter={(pid: string) => {
+              if (pid?.length > 0) {
+                return pid.replace(pid, pidFormatter(pid));
+              }
+              return '';
+            }}
+            field={withNameSpace('searchPid')}
+          />
+          <SearchButton
+            onClick={(e: any) => {
+              e.preventDefault();
+              handlePidChange(searchPid, nameSpace);
+            }}
+          />
+        </Form.Row>
+        <Form.Row>
+          <Label>PIN</Label>
+          <FastInput
+            formikProps={formikProps}
+            displayErrorTooltips
+            className="input-small"
+            disabled={false}
+            field={withNameSpace('searchPin')}
+            onBlurFormatter={(pin: number) => {
+              if (pin > 0) {
+                return pin;
+              }
+              return '';
+            }}
+            type="number"
+          />
+          <SearchButton
+            onClick={(e: any) => {
+              e.preventDefault();
+              handlePinChange(searchPin, nameSpace);
+            }}
+          />
+        </Form.Row>
+        <Form.Row>
+          <Label>Street Address</Label>
+          <GeocoderAutoComplete
+            value={searchAddress}
+            field={withNameSpace('searchAddress')}
+            onSelectionChanged={selection => {
+              formikProps.setFieldValue(withNameSpace('searchAddress'), selection.fullAddress);
+              setGeocoderResponse(selection);
+            }}
+            onTextChange={value => formikProps.setFieldValue(withNameSpace('searchAddress'), value)}
+            error={getIn(formikProps.errors, withNameSpace('searchAddress'))}
+            touch={getIn(formikProps.touched, withNameSpace('searchAddress'))}
+            displayErrorTooltips
+          />
+          <SearchButton
+            disabled={!geocoderResponse}
+            onClick={(e: any) => {
+              e.preventDefault();
+              geocoderResponse && handleGeocoderChanges(geocoderResponse, nameSpace);
+            }}
+          />
+        </Form.Row>
+      </Col>
+      <Col md={1}>
+        <h5>OR</h5>
+      </Col>
+      <Col md={5} className="instruction">
+        <p>
+          Click on the pin, and then click your desired location on the map to pull the parcel
+          details.
+        </p>
+        <Row>
+          <Col className="marker-svg">
+            <ClickAwayListener
+              onClickAway={() => {
+                setMovingPinNameSpace(undefined);
+              }}
+            >
+              <SearchMarkerButton
```

Comment:
Has the cursor idon been updated to the new icons? Right now in dev it's still the old one with the white dot

Reply:
I don't think I was given a new one for the search marker on the inventory forms.